lib/jsdom/living/events/MouseEvent-impl.js

```diff
@@ -16,31 +16,42 @@ class MouseEventImpl extends UIEventImpl {
   }
   get pageX() {
     if (this._dispatchFlag) {
-      return 0;
+      return this._initialContainingBlockRelativeY;
```

Comment:
should be `_initialContainingBlockRelativeX`, but the point is moot as I'm splitting this out into a separate PR